Cookbook.Application/Services/RecipebookWriteServices.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Cookbook.Application.Database;
+using Cookbook.Domain.Models;
+using FluentValidation;
+
+namespace Cookbook.Application.Services
+{
+    internal class RecipebookWriteServices : IRecipebookWriteServices
```

Comment:
Miert valasztottad szet a read-write service-t?

Reply:
a Clean codes video ajanlotta aztan jo otletnek tunt igy van 2 api es ha mondjuk van 1 felhasznalo akinek nem kene tudnia irnia akkor ha csak a read apit kapja meg akkor veletlen se tud valtoztatni. Vagy nem kene? Kozben rajottem h az implementaciot nincs ertleme szetvalasztani :D vagy te is azt kerdezed?

Reply:
igen igen igen. Az interface az jo otlet, eleg distinct responsibility-k meg lehet azt mondani hogy valaki esetleg csak olvasni akar, akkor kisebb "requirement" a read only interface, ez az I a SOLID-ban is, szoval az interface az okes, az implementacio lepett meg. Nem tudtam hogy "veletlen" vagy valahol olvastad hogy igy fasza